CONTRIBUTING.md

```diff
@@ -120,7 +120,6 @@ As discussed earlier, the operator-controller adheres to a microservice architec
 Unsure where to submit an issue?
 
 - [The Operator-Controller project](https://github.com/operator-framework/operator-controller/), which is the top level component allowing users to specify operators they'd like to install.
-- [The Catalogd project](https://github.com/operator-framework/catalogd/), which hosts operator content and helps users discover installable content.
 
 Don't worry if you accidentally submit an issue against the wrong project, if we notice that an issue would fit better with a separate project we'll move it to the correct repository and mention it in the #olm-dev slack channel.
```

Comment:
Can likely delete this, as there is just one place/project now.

Reply:
Done